Makefile

```
@@ -49,6 +49,13 @@ test: prepare ## Run all go-sdk tests
 .PHONY: integration
 integration: build-cli-cross-platform integration-only ## Build and run integration tests
 
+.PHONY: integration-generation
+integration-generation: build-cli-cross-platform integration-generation-only ## Build and run integration tests
+
+.PHONY: integration-generation-only
+integration-generation-only: ## Run integration tests
+	PATH=$(PWD)/bin:${PATH} go test -v github.com/lacework/go-sdk/integration -timeout 30m -run "^TestGeneration" -tags="generation"
```

Can we bump the version of gotestsum to v1.7.0 and use that here? Just adds a bit of color to the formatting which helps at a glance IMO.
```suggestion
	PATH=$(PWD)/bin:${PATH} gotestsum -- -v github.com/lacework/go-sdk/integration -timeout 30m -run "^TestGeneration" -tags="generation"
```

---

@rmoles seems reasonable to me; that said looks like we don't do this for the other integration tests so perhaps we should have a maint PR to bump version as well as change utility.  

@afiune @dmurray-lacework thoughts on this?

---

Sounds good!

---

Shall I fire a PR up to main with this change?

---

https://github.com/lacework/go-sdk/pull/632